docs/GettingStartedDocs/README.md

```diff
@@ -33,18 +33,12 @@ the hardware types below.
 
 ### Intel SGX
 
-If you would like to start developing apps with the preview Open Enclave SDK
-release, start here for instructions to install and use the SDK package:
-
-- [Ubuntu 18.04 with SGX hardware](install_oe_sdk-Ubuntu_18.04.md)
-- [Ubuntu 16.04 with SGX hardware](install_oe_sdk-Ubuntu_16.04.md)
-- [Ubuntu 18.04 or 16.04 in simulation mode](install_oe_sdk-Simulation.md)
```

Comment:
Should we just remove the 16.04 word there?

Reply:
@RRathna, I am sorry for any confusion, but the 1804 and simulation mode instructions can stay

Reply:
This is because someone who looks at our master branch should understand how to install OE SDK .

Reply:
Adding back steps to install OE SDK